Add style param to NAFC_page, make_ui_NAFC and trigger_button

fmhoeger opened the pull request.

pmcharrison commented:
Hi @fmhoeger , I'm happy to add this style argument. As you suggested, I think it's best if this is called `button_style` each time it occurs. 

Also, to ensure that the changes don't break pre-existing code, please could you move the new `button_style` argument so that it is the last in any argument list?

fmhoeger commented:
Hi @pmcharrison , I have now renamed the style argument to button_style in NAFC_page and make_ui_NAFC. In trigger_button I kept it as 'style' as it is already in the button context. Does this make sense? Also moved button_style to the last position in the argument lists.

pmcharrison commented:
Thanks @fmhoeger , all good! Merged.
